src/main/java/net/researchgate/azkaban/Group.java

```diff
@@ -0,0 +1,39 @@
+package net.researchgate.azkaban;
+
+import azkaban.user.Role;
```

Comment:
if your code has to be compliant with java 7 and you still would like to lambda-up some stuff .. you can add Google's Guava as dependency ... there are lots of cool stuffs to make your code look sexy ;)